src/librustc/session/config.rs

```diff
@@ -1342,10 +1342,15 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
     unpretty: Option<String> = (None, parse_unpretty, [UNTRACKED],
         "Present the input source, unstable (and less-pretty) variants;
         valid types are any of the types for `--pretty`, as well as:
-        `flowgraph=<nodeid>` (graphviz formatted flowgraph for node),
+        `expanded`, `expanded,identified`,
+        `expanded,hygiene` (with internal representations),
+        `flowgraph,unlabelled=<nodeid>` (unlabelled graphviz formatted flowgraph for node),
+        `flowgraph,unlabelled]` (unlabelled graphviz formatted flowgraph for node),
         `everybody_loops` (all function bodies replaced with `loop {}`),
-        `hir` (the HIR), `hir,identified`, or
-        `hir,typed` (HIR with types for each node)."),
+        `hir` (the HIR), `hir,identified`,
+        `hir,typed` (HIR with types for each node),
+        `hir-tree` (dump the raw HIR),
+        `mir` (the MIR), or `mir-cfg` (graphviz formatted MIR)"),
```

Comment:
```suggestion
        `mir` (the MIR), or `mir-cfg` (graphviz formatted MIR)"
    ),
```

sorry about the earlier suggestion, I didn't know about the restriction of the macro